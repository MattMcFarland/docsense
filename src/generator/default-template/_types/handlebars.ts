export interface BlockHelper {
  fn: (ctx: any) => any;
}
