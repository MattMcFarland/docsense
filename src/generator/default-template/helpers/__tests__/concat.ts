import * as Handlebars from 'handlebars';

describe('hbs helpers > concat', () => {
  test('matches snapshot', () => {
    Handlebars.registerHelper('concat', require('../concat').default);
    const hbs = `<div>{{concat foo " " bar " " baz}}</div>`;
    const template = Handlebars.compile(hbs);
    const result = template({ foo: 'foo', bar: 'bar', baz: 'baz' });
    expect(result).toMatchSnapshot();
  });
});
