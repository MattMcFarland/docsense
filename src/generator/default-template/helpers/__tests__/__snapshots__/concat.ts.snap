// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hbs helpers > concat matches snapshot 1`] = `"<div>foo bar baz</div>"`;
